schedule/yast/raid/raid1_sle_lvm.yaml

```diff
@@ -0,0 +1,36 @@
+---
+name:           lvm+raid1@64bit
+description:    >
+  Validation of partitioning for raid1 on lvm
+  Installation of RAID1 using expert partitioner.
+vars:
+  RAIDLEVEL: 1
+  LVM: 1
+test_data:
+  !include: schedule/yast/raid/testdata/lvm_raid1.yaml
+schedule:
+  - installation/isosize
+  - installation/bootloader
+  - installation/welcome
+  - installation/accept_license
+  - installation/scc_registration
+  - installation/addon_products_sle
+  - installation/system_role
+  - installation/partitioning
+  - installation/partitioning_raid
+  - installation/partitioning_finish
+  - installation/installer_timezone
+  - installation/hostname_inst
```

Comment:
Remove `installation/hostname_inst` due it is not essential for this test and we can reuse easily this yaml for other archs.

Reply:
done